src/main/java/lotto/domain/LottoNumber.java

```diff
@@ -0,0 +1,38 @@
+package lotto.domain;
+
+import java.util.Objects;
+
+public class LottoNumber {
+
+    private static final int MIN_NUMBER = 1;
+    private static final int MAX_NUMBER = 45;
+    private final int lottoNumber;
+
+    public LottoNumber(int lottoNumber) {
+        validateNumber(lottoNumber);
+        this.lottoNumber = lottoNumber;
+    }
+
+    private void validateNumber(int lottoNumber) {
+        if (lottoNumber < MIN_NUMBER || lottoNumber > MAX_NUMBER) {
```

Comment:
```suggestion
       // case 1
        if (MIN_NUMBER <= lottoNumber && lottoNumber <= MAX_NUMBER) {
       // case 2
        if (lottoNumber < MIN_NUMBER || MAX_NUMBER < lottoNumber) {
```
미세 가독성 팁인데 대조군이 가운데나 가장자리로 몰리면 가독성이 좋아집니다